### ISSUE TITLE
Task 3.III.a Add Dataset Description to Experimental Studies for the Second Proposed Methods of Task 3

### ISSUE THREAD

**Issue by Chlerry** [Chlerry/term-project#20]
Add descriptions to [README.md](https://github.com/Chlerry/term-project/blob/task3_doc/task3/README.md)
Dataset description: which datasets are used, how many training frames (validation set size if used), how many test frames, etc. Make sure that for different proposed networks, the training set should be the same, and the test set should also be the same. 

![image](https://user-images.githubusercontent.com/16151567/76281745-3ccb8580-6253-11ea-9ca7-95cfe882e97c.png)
